torch/distributed/distributed_c10d.py

```diff
@@ -1227,6 +1227,9 @@ def isend(tensor: torch.Tensor, dst: int, group: Optional[ProcessGroup] = None,
         Modifying ``tensor`` before the request completes causes undefined
         behavior.
 
+    .. warning::
+        ``tag`` and tensor ordering is not supported with the NCCL backend.
```

Comment:
Should "is" be "are"? 🤔

Reply:
updated to remove "tensor ordering" part